cmd/root.go

```diff
@@ -92,9 +92,9 @@ Complete documentation is available at https://appsody.dev`,
 		//Run: no run action for the root command
 	}
 
-	rootCmd.PersistentFlags().StringVar(&rootConfig.CfgFile, "config", "", "config file (default is $HOME/.appsody/.appsody.yaml)")
-	rootCmd.PersistentFlags().BoolVarP(&rootConfig.Verbose, "verbose", "v", false, "Turns on debug output and logging to a file in $HOME/.appsody/logs")
-	rootCmd.PersistentFlags().BoolVar(&rootConfig.Dryrun, "dryrun", false, "Turns on dry run mode")
+	rootCmd.PersistentFlags().StringVar(&rootConfig.CfgFile, "config", "", "Location of Appsody config files to use. (default '$HOME/.appsody/.appsody.yaml')")
+	rootCmd.PersistentFlags().BoolVarP(&rootConfig.Verbose, "verbose", "v", false, "Prints out more detailed logs on the console, and logs them to a file in $HOME/.appsody/logs.")
```

Comment:
Assuming the same thing is sent to the console and to the file, I suggest:

"Prints more detailed log output, to the console and to a file in $HOME/.appsody/logs."

Do we need to mention the name of the log file or will it be obvious, eg does it have a date/timestamp, or something to say which stack/app/whatever this log file came from?

Reply:
Yes, the log file will have the timestamp on it